src/components/Table/reelShadow.ts

```diff
@@ -0,0 +1,26 @@
+type ReelShadow = {
+  showShadow?: boolean
+  direction?: 'left' | 'right'
+}
+
+export const reelShadow = ({ showShadow = true, direction = 'left' }: ReelShadow) => {
+  const shadowWidth = '12px' //影の横幅はpx指定で固定とする
+
+  return `
+    &::after {
+      content: '';
+      position: absolute;
+      z-index: 0;
+      left: ${direction === 'left' ? '0' : `-${shadowWidth}`};
+      top: 0;
+      width: ${shadowWidth};
+      pointer-events: none; /* 影の領域が広すぎるとクリッカブルエリアを侵食するので無効化 */
+      inset-block: 0;
```

Comment:
- `inset-block` なので `top` が不要になる
- `left` と近い方がよさそう

```suggestion
      inset-block: 0;
      left: ${direction === 'left' ? '0' : `-${shadowWidth}`};
      width: ${shadowWidth};
      pointer-events: none; /* 影の領域が広すぎるとクリッカブルエリアを侵食するので無効化 */
```